MDSassignment/guestbook/post/forms.py

```diff
@@ -0,0 +1,25 @@
+from django import forms
+from django.forms import ModelForm
+from .models import Post
+
+
+class PostForm(forms.ModelForm):
+
+	class Meta:
+		model = Post
+		fields = ['name', 'text', 'email']
+
+
+	def clean_text(self):
```

Comment:
I like it. If I enter `100% free!` it's not spam anymore ;)

You probably want something like

```python
spam_strings = ['lose weight', '100% free', 'dear friend']
text_lower = text.lower()
for spam in spam_strings:
    if spam in text_lower:
        raise forms.ValidationError('fail: spam')
```